recipes/signxml/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "signxml" %}
+{% set version = "2.6.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "70e3edbb07b89bec94d39db2cdced724c540f9258366474177c746b9f903d9c4"
```

Could you please remove the quotes from ``sha256``?
```suggestion
  sha256: 70e3edbb07b89bec94d39db2cdced724c540f9258366474177c746b9f903d9c4
```